lms/djangoapps/instructor/management/commands/dump_userinfo.py

```diff
@@ -257,12 +262,20 @@ def handle(self, *args, **options):
             if 'Date Registered' not in student_dict:
                 student_dict['Date Registered'] = unpaid_registration_table[user_id].created
 
+            # self.add_fields_to adds fields to student_dict if they are not already present
+            # including student_dict['Certificate']
             certificate = self.add_fields_to(student_dict, CERTIFICATE_FIELDS, certificate_table, user_id)
 
-            # If the user has received a certificate, adjust their credits issued and course completion flag.
+            # If the student has hit the 'Grade Me' button.
```

Comment:
Maybe this should say: # Record whether student clicked the 'Grade Me' button?
(the current comment just feels incomplete, it doesn't explain what's being done as a consequence of students clicking the button)